lib/Runtime/Library/EngineInterfaceObject.h

```diff
@@ -123,7 +123,7 @@ namespace Js
 #define GlobalBuiltInConstructor(global)
 
 #define BuiltInRaiseException(exceptionType, exceptionID) \
-        static Var EntryIntl_BuiltIn_raise##exceptionID(RecyclableObject *function, CallInfo callInfo, ...);
+        static Var EntryBuiltIn_raise##exceptionID(RecyclableObject *function, CallInfo callInfo, ...);
```

Comment:
Let's use `Entry_` to match the convention used for other native entry points.

Reply:
Will do.